summary_statistics_analysis.py

```diff
@@ -13,8 +13,11 @@ def remove_phenos_from_analysis(mt: hl.MatrixTable):
     pass
 
 
-def annotate_mt_with_largest_meta_analysis(mt):
-    meta_mt = hl.read_matrix_table(get_meta_analysis_results_path())
+def annotate_mt_with_largest_meta_analysis(mt, h2_filter='none'):
+    if h2_filter == 'both':
```

Comment:
Is this only a problem if it's `both` or if it's anything except `none` or `pass`?

Reply:
load_meta_analysis_results throws an error if it's not `both`, `none`, or `pass`